tests/validation/tests/v3_api/resource/terraform/charts/test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+func main() {
+	installer := &releases.ExactVersion{
+		Product: product.Terraform,
+		Version: version.Must(version.NewVersion("1.0.6")),
```

Comment:
Any particular reason for specifying version 1.0.6? Wondering if it would be useful to define a version constraint instead, or add a parameter for the user to define the terraform version.